### ISSUE TITLE
[ZAuth] Make NATIVE_EITHER the default auth method

### ISSUE THREAD

**Issue by craigh** [zikula/core#4351]
NATIVE_EITHER should be the default method

**Comment by Guite** [comment]
- Also enable it for existing admins?
- Even enable it for all users with either `native_uname` or `native_email`?

**Comment by craigh** [comment]
@Guite your comments don't make sense to me. This request is to simply change the single default method of authentication used after installation. changing authentication methods after someone has registered probably isn't a good idea.

**Comment by Guite** [comment]
but we could think about changing it for UID 2 at least

**Comment by craigh** [comment]
changing auth methods on upgrade will break any accounts that do not use that method.

**Comment by Guite** [comment]
> maybe we should back this up and reconsider - maybe in the future, there should only be one native login method - EITHER.

**Comment by rallek** [comment]
From my point of view the login method is _native Zikula_. And with _native Zikula_ you can select if you allow login via email or uname or both.

**Comment by Kaik** [comment]
Changing login method from native_uname or native_email to NATIVE_EITHER is basically no brainer... 
Both as a default method and as a replacement for already selected native_uname or native_email for users.

**Comment by Kaik** [comment]
So I have forced change to use native_either for all accounts recently.
Below are sql that I used to manually change native method for all accounts.
`
UPDATE zauth_authentication_mapping SET method='native_either'
UPDATE users_attributes SET value='native_either' WHERE name LIKE 'authenticationMethod'
`
As I stated somewhere before, native_either is native_uname and native_email, it doesn't really matter from systems point of view. The only problem may occur only when multiple same email accounts are allowed.
